src/penn_chime/charts.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+from typing import Optional
```

Comment:
Lint says that this import isn't used anymore?

Reply:
removed